.github/minty.yaml

```diff
@@ -2,9 +2,14 @@ version: 'minty.abcxyz.dev/v2'
 
 rule:
   if: |-
-    assertion.iss == issuers.github &&
-    assertion.repository_owner_id == '93787867' &&
-    assertion.repository_id == '576289489'
+    (
+      assertion.iss == issuers.github &&
+      assertion.repository_owner_id == '93787867' &&
+      assertion.repository_id == '576289489'
+    ) || (
+      assertion.iss == issuers.google &&
+      assertion.sub == '112892967544648233217' # integration WIF service account
```

Comment:
I am not sure you can add comments inline CEL like this.....curious to see

Reply:
Yeah, that won't parse correctly:

```
[github-token-minter] [remotes/origin/dmearl/mintycfg-allow-google-token] [$] λ go run cmd/minty/main.go config validate --minty-file=.github/
minty.yaml


-- Error --
failed to compile CEL expressions in config: error compiling configuration ruleset: failed to compile ruleset: failed to compile CEL expression: ERROR: <input>:7:44: Syntax error: token recognition error at: '#'
 |   assertion.sub == '112892967544648233217' # integration WIF service account
 | ...........................................^
ERROR: <input>:7:46: Syntax error: mismatched input 'integration' expecting ')'
 |   assertion.sub == '112892967544648233217' # integration WIF service account
 | .............................................^

{"time":"2024-10-16T10:50:07.956415-04:00","severity":"ERROR","message":"process exited with error","error":"error executing validation: configuration failed to compile"}
exit status 1
```

Reply:
Fixed the inline CEL comments by using `//`. 

```
dmearl@dmearl-dev:~/code/abcxyz/github-token-minter$ go run cmd/minty/main.go config validate --minty-file=.github/minty.yaml

#################


Configuration and all rules compiled successfully
Config:
- Version: minty.abcxyz.dev/v2
- Rule.If: (
  assertion.iss == issuers.github &&
  assertion.repository_owner_id == '93787867' &&
  assertion.repository_id == '576289489'
) || (
  assertion.iss == issuers.google &&
  assertion.sub == '112892967544648233217' // integration WIF service account
)
- Scopes:
  - integ:
    - Rule.If: assertion.workflow_ref.startsWith("abcxyz/github-token-minter/.github/workflows/ci.yml") || (
  assertion.iss == issuers.google &&
  assertion.sub == "112892967544648233217" // integration WIF service account
)
    - Repositories: [github-token-minter]
    - Permissions: map[issues:read]
  - create-tag:
    - Rule.If: assertion.job_workflow_ref == "abcxyz/pkg/.github/workflows/create-tag.yml@refs/heads/main" &&
assertion.workflow_ref.startsWith("abcxyz/github-token-minter/.github/workflows/create-tag.yml") &&
assertion.ref == 'refs/heads/main' &&
assertion.event_name == 'workflow_dispatch'
    - Repositories: [github-token-minter]
    - Permissions: map[contents:write]

#################

Configuration parsed and loaded successfully

```